apps/service-portal/src/lib/masterNavigation.ts

```diff
@@ -31,20 +29,19 @@ export const rootNavigationItem: PortalNavigationItem = {
 export const MAIN_NAVIGATION: PortalNavigationItem = {
   ...rootNavigationItem,
   children: [
-    documentProviderNavigation,
-    documentsNavigation,
     applicationsNavigation,
-    personalInformationNavigation,
-    informationNavigation,
+    assetsNavigation,
     companyNavigation,
-    licenseNavigation,
+    delegationsNavigation,
+    documentProviderNavigation,
+    documentsNavigation,
     educationLicenseNavigation,
     educationNavigation,
-    accessControlNavigation,
-    assetsNavigation,
-    icelandicNamesRegistryNavigation,
     financeNavigation,
+    icelandicNamesRegistryNavigation,
+    informationNavigation,
+    licenseNavigation,
+    personalInformationNavigation,
     vehiclesNavigation,
-    delegationsNavigation,
```

Comment:
Even though I love having this alphabetically ordered, this has been ordered in the way in is supposed to appear in the service-portal menu 😄

Reply:
ahh snap sorry will revert :).
I was just discovering vs code sort alphabetically shortcut